unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldBridge/BuilderInWorldBridge.cs

```diff
@@ -28,13 +30,23 @@ public class BuilderInWorldBridge : MonoBehaviour
 
     StoreSceneStateEvent storeSceneState = new StoreSceneStateEvent();
     SaveSceneStateEvent saveSceneState = new SaveSceneStateEvent();
+    SaveSceneInfoEvent saveSceneInfo = new SaveSceneInfoEvent();
     ModifyEntityComponentEvent modifyEntityComponentEvent = new ModifyEntityComponentEvent();
     EntityPayload entityPayload = new EntityPayload();
     EntitySingleComponentPayload entitySingleComponentPayload = new EntitySingleComponentPayload();
     BuilderProjectPayload builderProjectPayload = new BuilderProjectPayload();
 
     #region MessagesFromKernel
 
+    public void SaveSceneInfo(ParcelScene scene, string sceneName, string sceneDescription, string sceneScreenshot)
+    {
+        saveSceneInfo.payload.title = sceneName;
+        saveSceneInfo.payload.description = sceneDescription;
+        saveSceneInfo.payload.screenshot = sceneScreenshot;
+
+        WebInterface.SendSceneEvent(scene.sceneData.id, BuilderInWorldSettings.STATE_EVENT_NAME, saveSceneInfo);
+    }
+
```

Comment:
This should be out of the region `MessagesFromKernel`

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/628/commits/363acada66c3cc2ae4a20a072ea251699aa997b4. Thanks!